data/mods/Magiclysm/enchanted/enchanted_wands.json

```diff
@@ -189,5 +222,353 @@
     "price": 30000,
     "description": "A slender wooden wand with a mana crystal socket at the base that casts a spell when activated.  This wand casts cone of cold.",
     "use_action": { "type": "cast_spell", "spell_id": "cone_cold", "no_fail": true, "level": 15, "need_wielding": true }
+  },
+  {
+    "type": "TOOL",
+    "id": "wand_chan_magic_missile_minor",
+    "name": "channeling minor wand of magic missile",
+    "name_plural": "channeling minor wands of magic missile",
+    "copy-from": "chan_wand",
+    "description": "A slender wooden wand with a mana channel at the base that casts a spell when activated.  This wand casts magic missile.",
+    "use_action": { "type": "cast_spell", "spell_id": "magic_missile", "no_fail": true, "level": 5, "need_wielding": true }
```

Comment:
if you're channeling it, why does it reduce the failure chance to 0? this makes these items much better than casting the spell at all.

Reply:
They are, yes. Basically what I did was create 3 tiers of wands, disposable, rechargeable, and channeling.
Channeling is at the top, it's just a wand that is designed and created solely to take mana you pump into it and put a spell out the other end, the downside is you're paying for it, which can be costly for say cone of cold.. the upside being it won't fail. What is being channeled is the mana/cost/power.
You are still holding a wand though and you can't cast other spells while doing so.
I try to think of magic items more like electronics, it's a magical circuit that performs the function it was designed to do. In this case, shoot magic missiles.
As far as it being much better than casting the spell yourself, this is why we created spears, bows, slings, and guns. Tools do a better job than just using your hands.

Reply:
hm, i request that you separate out the channeling wands from this PR to be discussed later. This is a pretty severe balance decision to be making, and i didn't see anything wrong with the other wands.